### ISSUE TITLE
User documentation

### ISSUE THREAD

**Issue by davidmehren** [hedgedoc/hedgedoc#713]
**Which part of the project should be enhanced?**
Documentation

**Is your enhancement request related to a problem? Please describe.**
The current docs on docs.hedgedoc.org only contain information for administrators.
Users are expected to find out how HedgeDoc works by reading the `feature.md` and clicking around.

**Describe the solution you'd like**
A separate user documentation, that describes (atleast) the following things:
- render/markup features: it's probably best to just direct users to the features page, as they then can see the features directly in action
- slide mode
- profile page, login
- history, tags, search, export, import

**Describe alternatives you've considered**
leaving users clueless ;)

**Additional context**
Other projects also have a separate user documentation:
- Nextcloud has [user](https://docs.nextcloud.com/server/latest/user_manual/en/), [admin](https://docs.nextcloud.com/server/20/admin_manual/) and [development](https://docs.nextcloud.com/server/latest/developer_manual/) manuals.
- Zammad has [install](https://docs.zammad.org/en/latest/), [admin](https://admin-docs.zammad.org/en/latest/) and [user](https://user-docs.zammad.org/en/latest/) docs.

**Comment by DerMolly** [comment]
I'd suggest explaining the editor interface, too. It is the interface users will spend most of their time…

**Comment by DerMolly** [comment]
Also slide mode has it's own file in the release see <https://demo.hedgedoc.org/slide-example>

**Comment by SISheogorath** [comment]
We should try to make sure that the user documentation is available on a HedgeDoc instance. This reduces the dependency on our online documentation and provides us with more flexibility. It also ensures that users automatically use the right documentation for the version that is installed on their setup as well as allowing forks to adjust/translate documentation easier.

**Comment by DerMolly** [comment]
I agree with the features, slide and yaml pages, but explaining the interface seems to really benefit from a multi-page approach something that we can't easily provide on the instance. (note names need to be blocked for those pages, that might be already used in an upgrading instance…)
Our Documentation seemed to be a good place for something like this…

**Comment by aschrijver** [comment]
Nice to see this issue. I love HedgeDoc, have it installed via Yunohost. But there's a bunch of things in the UI I find super confusing:

- There's a "Clear History" that seems like a high danger click, as I will never know my pads again if they aren't listed anymore.
- I told a user "Please delete that pad" with response "I don't know how". And as admin I don't know how either. (Somewhat relates to #1547)
- In my profile dropdown there's a "Delete user" entry, which seems to be another high-danger action I should avoid clicking at all cost.
- An Admin CLI and Admin GUI is in the works, which is great. Right now there's no insight and guest users could maintain pads indexing the entire Dark Web without me knowing about it.

These things make me feel less secure and HedgeDoc less robust, like I need to treat this app with delicate care or I will booby trap myself or others.

**Comment by archandrogyne** [comment]
On the page "supported yaml metadata" it says to format like this at the start of the doc:

---
YAML metas
---

But this does not work, and you have to do this instead:

---
YAML metas

---

Not huge, but would be better to add the extra line to the doc to reduce confusion
